scipy/special/_erfinv.h

```diff
@@ -0,0 +1,62 @@
+#ifndef ERF_INV_H
+#define ERF_INV_H
+
+/*
+ * mconf configures NANS, INFINITYs etc. for cephes and includes some standard
+ * headers. Although erfinv and erfcinv are not defined in cephes, erf and erfc
+ * are. We want to keep the behaviour consistent for the inverse functions and
+ * so need to include mconf.
+ */
+#include "cephes/mconf.h"
+
+/*
+ * Inverse of the error function.
+ *
+ * Computes the inverse of the error function on the restricted domain
+ * -1 < y < 1. This restriction ensures the existence of a unique result
+ * such that erf(erfinv(y)) = y.
+ */
+double erfinv(double y) {
+    if (cephes_isnan(y)) {
+        sf_error("erfinv", SF_ERROR_DOMAIN, NULL);
+        return NPY_NAN;
```

Comment:
Small thing: when `y` is NaN you generally want to return `y` here. This is because there are actually many NaNs in IEEE double precision and you want the particular NaN you received to propagate if possible.

Reply:
Done!